lib/sda/service.go

```diff
@@ -295,7 +297,7 @@ in place of the standard reply. The Client.Send method will catch that and retur
 
 // Client for a service
 type Client struct {
-	private abstract.Secret
+	Private abstract.Secret
```

Comment:
https://github.com/dedis/cothority/pull/404/files#diff-6510846677b7d460f30e2726622e9869R183

and when saving the client-structure it's useful to have that field public. Else we'll have to write `MarshalBinary`-methods.